app/bundles/ConfigBundle/Model/SysinfoModel.php

```diff
@@ -59,7 +67,7 @@ public function getPhpInfo()
             return $this->phpInfo;
         }
 
-        if (function_exists('phpinfo')) {
+        if (function_exists('phpinfo') && 'cli' !== php_sapi_name()) {
```

Comment:
This is needed because when PHP is running on the CLI (e.g. during automated tests), `phpinfo()` doesn't render HTML but regular text + line breaks instead. Then the logic within this `if` block breaks. This doesn't change any existing behavior for "normal" users, and the formatting is updated in the UI as it was before.